### DISCUSSION TITLE
how to create a file to load into playback window?

### DISCUSSION THREAD

**Discussion by dieselnutjob** [collin80/SavvyCAN#663] [General]
Hi
In the instructions under "Playback window" it says "A playback item can either come from a file (Load File) or...", but, I can't find any instructions on how to create such a file.
When I was capturing frames I remember two options to save to text or csv.
The csv file doesn't seem to have any payload, so I guess that the text option is the one to go for if I want to play them back?

Here is one of my saved files.

`Time: 150.537702    ID: 0x1F8 Std Bus: 0 Len: 8
Data Bytes: 0x00 0x00 0x00 0x00 0xFE 0xFF 0x00 0x00 

Time: 150.530895    ID: 0x316 Std Bus: 0 Len: 8
Data Bytes: 0x01 0x43 0xD3 0x15 0x43 0x00 0x00 0x00 

Time: 150.531899    ID: 0x329 Std Bus: 0 Len: 8
Data Bytes: 0x60 0x9C 0x00 0x00 0x00 0x00 0x00 0x00 

Time: 150.537414    ID: 0x1F0 Std Bus: 0 Len: 8
Data Bytes: 0x0A 0x00 0x0A 0x00 0x0A 0x00 0x0A 0x00 

Time: 150.446395    ID: 0x618 Std Bus: 0 Len: 5
Data Bytes: 0x00 0x0F 0x00 0x00 0x00 

Time: 150.446270    ID: 0x615 Std Bus: 0 Len: 5
Data Bytes: 0x07 0x00 0x00 0x00 0x00 

Time: 150.537352    ID: 0x153 Std Bus: 0 Len: 8
Data Bytes: 0x00 0x00 0x00 0xFF 0x00 0xFE 0xFF 0x00 

Time: 150.446041    ID: 0x613 Std Bus: 0 Len: 5
Data Bytes: 0x91 0x3C 0x1F 0x9F 0x4E 

Time: 150.532926    ID: 0x545 Std Bus: 0 Len: 8
Data Bytes: 0x00 0x05 0x48 0x00 0x00 0x00 0x00 0x00 
`

is this the right format to load back into the Playback window?

**Comment by collin80** [comment]
Nope, that's not the right format. On the main screen until the file menu the third entry down should be "Save Log File". You capture traffic then use that to save it to file. From there you can load it again any time by using the first entry in the file menu "Load Log File" or you can load it in the playback window to play it back on a CAN bus.

**Reply by dieselnutjob** [reply]
thanks that worked.
Is it possible to save a log file of just the last packets (the ones visible on the screen) when viewing packets in overwrite mode?
It's a lot easier to edit a file to make a playback file when there's just a few entries rather than thousands of lines.
